docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -5,17 +5,17 @@ rss_href: /feed_rss_created.xml
 
 # New Coverage engine status checks November 23rd, 2023
 
-!!! info "This release note applies if you have set Codacy to send pull request status data to your Git provider (see how on [GitHub](../../repositories-configure/integrations/github-integration.md#status-checks), [GitLab](../../repositories-configure/integrations/gitlab-integration.md#pull-request-status), and [Bitbucket](../../repositories-configure/integrations/bitbucket-integration.md#pull-request-status))."
+!!! info "This release note applies if you have set Codacy to send pull request status data to your Git provider (see how on [GitHub](../../repositories-configure/integrations/github-integration.md#status-checks), [GitLab](../../repositories-configure/integrations/gitlab-integration.md#pull-request-status), and [Bitbucket](../../repositories-configure/integrations/bitbucket-integration.md#pull-request-status))."<!--NOTE Temporary comment for review purpose, to remove-->
 
 On November 23rd 2023 we activated the new, faster Coverage engine and set it to send coverage data to your Git provider. This is part of an ongoing effort to improve the speed and value of the insights provided by Codacy.
 
-As a consequence of this update, you can now see two additional checks on your pull requests, marked **beta**, alongside the preexisting Codacy Coverage check. If you are using the Codacy Coverage status check to block merging pull requests on GitHub, please [update your setup](#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github) and avoid disruptions in the future.
+As a consequence of this update, you can now see two additional checks on your pull requests, marked **beta**, alongside the preexisting Codacy Coverage check. If you are using the Codacy Coverage status check to block merging pull requests on GitHub, please [update your setup](#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github) and avoid disruptions in the future.<!--NOTE Temporary comment for review purpose, to remove-->
 
 This is what the new checks look like on each provider:
 
 -   On GitHub:
 
-    ![New Coverage status checks GitHub](../images/ala-695-status-checks-github.png)
+    ![New Coverage status checks GitHub](../images/ala-695-status-checks-github.png)<!--NOTE Temporary comment for review purpose, to remove-->
```

Comment:
Although it would be nice, it's a minor improvement requiring a lot of effort to recreate the screenshots (especially on GL and BB). I suggest skipping this one 🤔